examples/multisource_adapt/main.py

```diff
@@ -0,0 +1,82 @@
+"""This example is about domain adaptation for digit image datasets, using PyTorch Lightning.
+
+Reference: https://github.com/thuml/CDAN/blob/master/pytorch/train_image.py
+"""
+
+import argparse
+import logging
+import os
+import sys
+
+import pytorch_lightning as pl
+from config import get_cfg_defaults
+from model import get_model
+from pytorch_lightning.callbacks import ModelCheckpoint
+from pytorch_lightning.loggers import TensorBoardLogger
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "../..")))
```

Comment:
is this still necessary? we stopped using this long time ago. See digits example.